ConfigManager.cs

```diff
@@ -24,8 +25,8 @@ public class Config
 		}
 		catch
```

Comment:
Provide more information

```suggestion
		catch (Exception e)
```